packages/transform/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@googleforcreators/transform",
```

Comment:
This file should use spaces, not tabs.

Seems like we're not properly linting that 🤔 

Maybe `npx prettier --write packages/transform/package.json` fixes it automatically?

Reply:
`prettier --write` is already in place for `lint-staged` but  `package.json` was ignored for prettier in https://github.com/GoogleForCreators/web-stories-wp/commit/177ec3bb91c9bbae85b066ee499101f0ec9719d5

Reply:
Ah, then I‘ll make this a todo for myself to look into for the future!